commercetools-models/src/main/java/io/sphere/sdk/products/commands/ProductImageUploadCommand.java

```diff
@@ -0,0 +1,104 @@
+package io.sphere.sdk.products.commands;
+
+import com.fasterxml.jackson.databind.JavaType;
+import io.sphere.sdk.client.HttpRequestIntent;
+import io.sphere.sdk.commands.CommandImpl;
+import io.sphere.sdk.http.HttpMethod;
+import io.sphere.sdk.http.UrlQueryBuilder;
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.products.ByIdVariantIdentifier;
+import io.sphere.sdk.products.Product;
+
+import javax.annotation.Nullable;
+import java.io.File;
+
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+public final class ProductImageUploadCommand extends CommandImpl<Product> {
```

Comment:
May be we can add some parts of our platform documentation http://dev.commercetools.com/http-api-projects-products.html#upload-a-product-image here? I would suggest to at least document which image formats are supported.